scripts/qa_crowbarsetup.sh

```diff
@@ -853,9 +853,9 @@ function onadmin_prepare_cloud_repos
     mkdir -p ${targetdir}
 
     if [ -n "${localreposdir_target}" ]; then
-        add_bind_mount \
-            "${localreposdir_target}/${CLOUDLOCALREPOS}/sle-12-$arch/" \
-            "${targetdir}"
+        add_mount "${CLOUDLOCALREPOS}" \
+                "$nfsserver_ip:$nfsserver_base_path/repos/$arch/${CLOUDLOCALREPOS}/" \
+                "$targetdir"
```

Comment:
in case of localreposdir it uses 9p for mounting from the mkcloud host instead of nfs (1st parameter iirc instead of 2nd)